### PR TITLE
[sc-8537]: sign helm release artifacts

### PR DESCRIPTION
**Pull request by rnts08**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -27,23 +27,22 @@ jobs:
...
+          keyring="$gpg_dir/secring.gpg"
+          base64 -d <<< "$GPG_HELM_SIGNING_KEY_BASE64" > "$keyring"
+
+          passphrase_file="$gpg_dir/passphrase"
+          echo "$GPG_HELM_SIGNING_PASS" > "$passphrase_file"
+
+          echo "CR_PASSPHRASE_FILE=$passphrase_file" >> "$GITHUB_ENV"
+          echo "CR_KEYRING=$keyring" >> "$GITHUB_ENV"
+        env:
+          GPG_KEYRING_BASE64: "${{ secrets.GPG_KEYRING_BASE64 }}"
```
Comment (WesleyCharlesBlake) [inline .github/workflows/release.yaml:44]: These need toatch the secrets naming (ie the github secrets we created)

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -27,23 +27,22 @@ jobs:
...
+          GPG_PASSPHRASE: "${{ secrets.GPG_PASSPHRASE }}"
```
Comment (WesleyCharlesBlake) [inline .github/workflows/release.yaml:45]: These need toatch the secrets naming (ie the github secrets we created)

**Review by WesleyCharlesBlake — CHANGES_REQUESTED** [review]
Just need to fix the secrets references

File: .github/workflows/release.yaml (hunk 1 above)
Comment (rnts08) [reply .github/workflows/release.yaml]: Switched the env variables to the right side.

**Review by WesleyCharlesBlake — APPROVED** [review]
LGTM

**Pull request merged**